phoenixRest/tests/views/card_order/__init__.py

```diff
@@ -0,0 +1,81 @@
+from test_app import TestApp
+
+def test_card_order(testapp:TestApp):
+    # Log in as an admin user
+    admin_user_token, refresh = testapp.auth_get_tokens("test@example.com", "sixcharacters")
+    admin = testapp.get_user(admin_user_token)
+    admin_uuid = admin["uuid"]
+    
+    # We ensure that admin has an avatar
+    testapp.upload_avatar("test@example.com", "sixcharacters", "phoenixRest/tests/assets/avatar_test.png", 10, 10, 600, 450, expected_status=200)
+    
+    #? ---- __init__.py ----
+    # Admin orders a card for themself
+    res = testapp.post_json("/card_order/", dict({
+        "user_uuid" : admin_uuid
+    }), headers=dict({
+        'Authorization': "Bearer " + admin_user_token
+    }), status=200)
+    
+    import uuid
+    # Admin orders a card with the wrong user uuid
+    res = testapp.post_json("/card_order/", dict({
+        "user_uuid" : str(uuid.uuid4())
+    }), headers=dict({
+        'Authorization': "Bearer " + admin_user_token
+    }), status=400)
+    
+    # Admin views all card orders i.e the one they created
+    res = testapp.get("/card_order/", headers=dict({
+        'Authorization': "Bearer " + admin_user_token
+    }), status=200).json_body
+    assert len(res) == 1
+    
+    #? ---- instance.py ----
+    # We get the uuid of the created order
+    card_order_uuid = res[0]["uuid"]
+    
+    from phoenixRest.models.crew.card_order import OrderStates
```

Comment:
again why import in function body? this smells of chatGPT code :D

Reply:
It's not lmao, i was just in a silly mood ;)

Reply:
the comments made me think so as well. It looked too professional lmao